sqsdomain/routable_pool.go

```diff
@@ -33,9 +33,6 @@ type RoutablePool interface {
 	CalculateTokenOutByTokenIn(ctx context.Context, tokenIn sdk.Coin) (sdk.Coin, error)
 	ChargeTakerFeeExactIn(tokenIn sdk.Coin) (tokenInAfterFee sdk.Coin)
 
-	// SetTokenOutDenom sets the token out denom on the routable pool.
-	SetTokenOutDenom(tokenOutDenom string)
```

Comment:
Can we remove this at a later date? Will this break the node, if `sqsdomain` isn't updated?

Reply:
No, this will not break the node.

It would only break if we changed serialization.

Reply:
Great, happy to approve then